src/bors/command/parser.rs

```diff
@@ -321,6 +323,30 @@ fn parse_priority_arg<'a>(parts: &[CommandPart<'a>]) -> Result<Option<u32>, Comm
     Ok(priority)
 }
 
+fn parse_tree_open<'a>(command: &'a str, _parts: &[CommandPart<'a>]) -> ParseResult<'a> {
```

Comment:
Let's use the original [homu](https://bors.rust-lang.org/) commands, so `treeclosed-` for opening a tree.